truss/patch/local_truss_patch_applier.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import subprocess
+from pathlib import Path
+
+from truss.templates.control.control.helpers.errors import UnsupportedPatch
+from truss.templates.control.control.helpers.truss_patch.model_code_patch_applier import (
+    apply_model_code_patch,
+)
+from truss.templates.control.control.helpers.types import (
+    Action,
+    ModelCodePatch,
+    Patch,
+    PythonRequirementPatch,
+    SystemPackagePatch,
+)
+from truss.truss_config import TrussConfig
+
+
+class LocalTrussPatchApplier:
```

Comment:
Is it worth subclassing `ModelContainerPatchApplier` or even using the same class instead of having a separate implementation here?

Reply:
Yes – I think we now have 3 PatchAppliers, I can make a base class they all inherit from.

Reply:
Let me take this on in a follow up